vita3k/emuenv/include/emuenv/state.h

```diff
@@ -122,7 +122,7 @@ struct EmuEnvState {
     uint32_t avg_fps = 0;
     uint32_t min_fps = 0;
     uint32_t max_fps = 0;
-    float fps_values[20] = {};
+    float fps_values[60] = {};
```

Comment:
no touch on this, this is used for can store last 20 fps number, for can display graphics in level max, corresponding 20 sec